src/framework/uicomponents/qml/Muse/UiComponents/StyledGroupBox.qml

```diff
@@ -32,11 +32,10 @@ GroupBox {
     spacing: 4
 
     background: Rectangle {
-        x: root.leftInset
-        y: root.topInset + root.implicitLabelHeight + root.spacing
+        y: root.topPadding - root.padding
 
-        width: root.width - x - root.rightInset
-        height: root.height - y - root.bottomInset
+        width: parent.width
+        height: root.height - y
```

Comment:
This doesn't immediately look right to me. What problem does it solve?
`root.topPadding - root.padding` seems a strange value, since most of the time it will be `0`, meaning that the `background` will overlap the `label`. Also, note the different between `inset` and `padding`: I believe `inset` is supposed to be the space _outside_ the `background` (which is almost always `0`), while `padding` is the space inside the `background`, between the border and the content.

Reply:
Maybe changing only `y` to 
```qml
        y: root.label && root.label.visible 
           ? root.topInset + root.implicitLabelHeight + root.spacing
           : root.topInset
```
works?